invenio_app_ils/ui/src/invenio_app_ils/common/api/eitems/eitem.js

```diff
@@ -46,10 +46,29 @@ const queryBuilder = () => {
   return new QueryBuilder();
 };
 
+const count = query => {
+  return http.get(`${eitemURL}?q=${query}`).then(response => {
+    response.data = response.data.hits.total;
```

Comment:
Instead of overriding the data of the response, its is more preferable to handle it by looking for total wherever you use it.